packages/wp-now/README-new.md

```diff
@@ -0,0 +1,140 @@
+-   `--wp`: the version of WordPress to use. This is optional and if not provided, it will use a default version. The default version is set to the [latest WordPress version](https://wordpress.org/download/releases/)(example usage: `--wp=5.8`)
+
+## Technical Details
+
+-   The `~/.wp-now` home directory is used to store the WP versions and the `wp-content` folders for projects using theme and plugin mode. The path to `wp-content` directory for the `plugin` and `theme` modes is `~/.wp-now/wp-content/${projectName}`.
+-   For the database setup, `wp-now` is using [Sqlite database integration plugin](https://wordpress.org/plugins/sqlite-database-integration/). The path to Sqlite database is ` ~/.wp-now/wp-content/${projectName}/database/.ht.sqlite`
+
+## Known Issues
+
+-   Running `wp-now start` in 'wp-content' or 'wordpress' mode will produce some empty directories: [WordPress/wordpress-playground#328](https://github.com/WordPress/wordpress-playground/issues/328)
+-   If you have an existing MySQL database defined in your `wp-config.php`, `wp-now` will still mount SQLite and the site won't load: [WordPress/wordpress-playground#327](https://github.com/WordPress/wordpress-playground/issues/327)
+
+## Comparisions
+
+### Laravel Valet
+
+If you are migrating from Laravel Valet, you should be aware of the differences it has with `wp-now`:
+
+-   `wp-now` does not require you to install WordPress separately, create a database, connect WordPress to that database or create a user account. All of these steps are handled by the `wp now start` command and are running under the hood;
+-   `wp-now` works across all platforms (Mac, Linux, Windows);
+-   `wp-now` does not support custom domains or SSL (yet!);
+-   `wp-now` works with WordPress themes and plugins even if you don't have WordPress installed;
+-   `wp-now` allows to easily switch the WordPress version with `wp-now start --wp=version.number`(make sure to replace the `version.number` with the actual WordPress version);
+-   `wp-now` does not support Xdebug PHP extension (yet!)
+
+Some similarities between Laravel Valet and `wp-now` to be aware of:
+
+-   could be used for non-WordPress projects;
+-   deployments are not possible with neither Laravel Valet, nor `wp-now`;
+-   possible to switch easily the PHP version;
+-   possibility to work on multiple WordPress sites simultaneously
+
+### wp-env
+
+If you are migrating from `wp-env`, you should be aware of the differences it has with `wp-now`:
+
+-   `wp-now` supports non-WordPress projects;
+-   `wp-now` does not need Docker;
+-   `wp-now` does not support Xdebug PHP extension (yet!);
+-   `wp-now` does not include Jest for automatic browser testing
+
+Some similarities between `wp-env` and `wp-now` to be aware of:
+
+-   no support for custom domains or SSL;
+-   `plugin`, `themes` and index modes are available on `wp-env` and `wp-now`;
+-   deployments are not possible with neither `wp-env`, nor `wp-now`;
```

Comment:
Do we need to mention that? Do users expect that local environment tool should handle deployments?

Reply:
I don't think there's harm in including it. Some users might expect this.